### DISCUSSION TITLE
Descriptions for codes with NDCs

### DISCUSSION THREAD

**Discussion by lvelari** [CMSgov/hospital-price-transparency#58] [Q&A] [answered]
Hello,

Our team is having a bit of an issue that we are hoping to get clarification on.

Currently our descriptions are based off of CPT/HCPCS. For drugs this can get a bit complicated.

For example:
HCPC J2700 has a CDM description of "OXACILLIN INJ UP TO 250MG"

However, we have multiple service code/NDC combinations for that same CPT that coincide with different charges - currently we have the description based off of the CPT for all of them. Our pharmacy has these descriptions labeled differently based on NDC, which seem to fit better with the charges and would give more information/transparency than the CDM description.

For codes that have NDCs, should we be using the description based off of the CPT/HCPCS or the NDC? We were leaning towards using the one that would describe the charges the best (NDC) but wanted to get more information before we continue.

Thank you!

**Comment by shubham763** [comment]
The decision of whether to use descriptions based on CPT/HCPCS codes or NDCs depends on the specific requirements and priorities of your organization, as well as regulatory considerations. Ultimately, choosing descriptions that accurately reflect the nature of the services provided and support efficient billing and reimbursement processes is essential. Consulting with relevant stakeholders, including billing specialists, clinicians, and regulatory experts, can help ensure that your documentation practices align with best practices and compliance requirements.

**Comment by RidenourK** [comment, marked as answer]
@lvelari. Thank you for your question. 

As indicated at [45 CFR 180.50(b)(2)(iii)(C)](https://www.ecfr.gov/current/title-45/part-180#p-180.50(b)(2)(iii)(C)), each hospital must encode in its machine-readable file all standard charge information, as applicable, for each of the following required data elements…**A description of the item or service that corresponds to the standard charge** established by the hospital, including: Beginning January 1, 2025, for drugs, the drug unit and type of measurement.
